test/studies/labelprop/labelprop-tweets.chpl

```diff
@@ -208,8 +208,10 @@ proc process_json(logfile:channel, fname:string, ref Pairs) {
     } catch e: SystemError {
       if e.err == EEOF then break;
 
-      stderr.writeln("severe error reading tweets ", fname, " offset ",
+      try! {
+        stderr.writeln("severe error reading tweets ", fname, " offset ",
           logfile.offset(), " : ", errorToString(e.err));
+      }
```

Comment:
This `try!` was needed due to the bug described in #7130, and should be removed when that is closed.